auto-insurance-claims-agentic-RAG/app.py

```
@@ -0,0 +1,136 @@
+import streamlit as st
+import json
+import os
+from insurance_claim_processor import process_claim, ClaimInfo
```

Ruff (0.8.2)

`insurance_claim_processor.ClaimInfo` imported but unused

Remove unused import: `insurance_claim_processor.ClaimInfo`

(F401)